blocks/library/text/index.js

```diff
@@ -47,7 +47,25 @@ registerBlock( 'core/text', {
 		}
 	],
 
-	edit( { attributes, setAttributes, insertBlockAfter, focus, setFocus } ) {
+	formatting: [
+		{
+			icon: 'editor-bold',
+			title: wp.i18n.__( 'Bold' ),
+			format: 'bold'
+		},
+		{
+			icon: 'editor-italic',
+			title: wp.i18n.__( 'Italic' ),
+			format: 'italic'
+		},
+		{
+			icon: 'editor-strikethrough',
+			title: wp.i18n.__( 'Strikethrough' ),
+			format: 'strikethrough'
+		}
+	],
+
+	edit( { attributes, setAttributes, insertBlockAfter, focus, setFocus, onFormatChange, formats } ) {
```

Comment:
Same here, also since these are probably things the block implementor doesn't need to know anything about.